pkg/client/client.go

```diff
@@ -217,7 +218,8 @@ func newControllerCliOptions() controllercli.Options {
 	mapper.Add(olmapiv1alpha1.SchemeGroupVersion.WithKind(meta.KindSubscription.Name), &restScope{name: apimeta.RESTScopeNameNamespace})
 	mapper.Add(monv1.SchemeGroupVersion.WithKind(meta.KindPrometheus.Name), &restScope{name: apimeta.RESTScopeNameNamespace})
 	mapper.Add(corev1.SchemeGroupVersion.WithKind(meta.KindPod.Name), &restScope{name: apimeta.RESTScopeNameNamespace})
-	mapper.Add(apibuildv1.SchemeGroupVersion.WithKind(meta.KindBuildConfig.Name), &restScope{name: apimeta.RESTScopeNameNamespace})
+	mapper.Add(apibuildv1.GroupVersion.WithKind(meta.KindBuildConfig.Name), &restScope{name: apimeta.RESTScopeNameNamespace})
+	mapper.Add(infinispanv1.SchemeGroupVersion.WithKind(meta.KindInfinispan.Name), &restScope{name: apimeta.RESTScopeNameNamespace})
```

Comment:
Is it needed?

Reply:
Yes, it is needed, otherwise the client cannot properly marshall/unmarshall payloads. Moreover, SchemeGroupVersion of BuildConfig is now deprecated so I changed it to GroupVersion as written in documentation.

Reply:
You can remove the Infinispan mapper from you PR, it was already added as part of different PR, see https://github.com/kiegroup/kogito-cloud-operator/blob/master/pkg/client/client.go#L223

Reply:
Yep, that I know. But thanks for a reminder :)